### PR TITLE
HomePage Component Added

### PR DESCRIPTION
**Pull request by ShekharDhangar**
HomePage Component Added. 
Review Pages and Component Folder. 
Styles are almost same as previous Ecommerce.

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            <li key={links.id} className="footer-link">
+              {links.title}
+            </li>
+          ))}
+      </ul>
+
+      <ul className="list-none footer-ul p-1">
+        <li className="list-head bold">SHED</li>
+        {footerData
+          .filter((links) => links.key === "shed")
```
Comment (hsnice16) [inline src/components/Footer/Footer.jsx:20]: we can improve the code here a little bit more, like you are comparing only `key` property so, you can use restructuring here.

Something like, 
```javascript
.filter(({key}) => key === "shed")
```

And for the `map`
```javascript
.map(({id, title}) => ...then use)
```

File: src/components/Footer/footerData.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+const footerData = [
```
Comment (hsnice16) [inline src/components/Footer/footerData.js:1]: This is nice, you have maintained all the footer data at one place. In this way, you can change footer data easily.

**Review by hsnice16 — APPROVED** [review]
Nice work. The code quality is good. Just added a few suggestions.

**Review by vrx29 — APPROVED** [review]
Nice work. your code looks clean

File: src/components/Footer/Footer.jsx (hunk 1 above)
Comment (ShekharDhangar) [reply src/components/Footer/Footer.jsx:20]: Got it, but some issue is going with my github desktop in this branch. So the said changes are been made and raised in my next PR

**Pull request merged**